manifests/pki/certificate.pp

```diff
@@ -34,7 +39,6 @@
     command => "${strongswan::pki::ca::pki_command} pki --gen --type rsa --size 2048 --outform der > ${private_key_dir}/${title}.der",
     creates => [ "${private_key_dir}/${title}.der"],
     path    => ['/usr/bin', '/usr/sbin'],
-    require => Class['strongswan::pki::ca'],
```

Comment:
Instead of having the first exec in a chain of resources having this dependency, the whole defined type starts with `require strongswan::pki::ca` (which also means `$ca_name = $strongswan::pki::ca::ca_name` above works)